internal/pkg/template/diff/override.go

```diff
@@ -34,18 +36,65 @@ func (m *ignorer) match(_, _ *yaml.Node, key string) bool {
 }
 
 // Parse is a no-op for an ignorer.
-func (m *ignorer) parse(_, _ *yaml.Node, _ string) (diffNode, error) {
+func (m *ignorer) parse(_, _ *yaml.Node, _ string, _ overrider) (diffNode, error) {
 	return nil, nil
 }
 
-type noopOverrider struct{}
+// intrinsicFuncFullShortFormConverter handles comparison between full/short form of an intrinsic function.
+type intrinsicFuncFullShortFormConverter struct{}
```

Comment:
will this ignore value changes? for example:
```
# from
!Ref abc
# to
Ref: xyz
```
this doesn't get ignored, right?

Reply:
Nope! I'm pushing tests to show this